### ISSUE TITLE
Help us and other developers by writing a blog post about pysonDB

### ISSUE THREAD

**Issue by Adwaith-Rajesh** [fredysomy/pysonDB#55]
Hey guys, if you like how pysondb works, and want other people to know about us, then write a blog post about using pysondb in you projects and you user experience  and spread the word.
One you have done. Share the link here. 

##  Let's take pysondb to the moon 🚀🚀🚀🚀

I got the idea for this issue from this [post](https://www.linkedin.com/posts/eldadfux_hacktoberfest-opensource-activity-6850188282857299968-n-8o)

**Comment by jakbin** [comment]
i think that we should wright blog post for this into 'docs' folder.

**Comment by Adwaith-Rajesh** [comment]
@jakbin I didn't get you dude

**Comment by Adwaith-Rajesh** [comment]
are you saying like in the docs create page mentioning this issue ??

**Comment by jakbin** [comment]
yes

**Comment by Adwaith-Rajesh** [comment]
@fredysomy

**Comment by jakbin** [comment]
And change github-pages folder to 'docs'.

**Comment by fredysomy** [comment]
I dont think we need to have this in the docs page and a blog is never related in any means with docs (with blogs i mean, something like intro, something like a project description).

**Comment by jakbin** [comment]
ok
